.rspec

```diff
@@ -0,0 +1,3 @@
+--format documentation
+--color
+--require spec_helper
```

Comment:
As mentioned in the [RSpec documentation](https://relishapp.com/rspec/rspec-core/v/3-5/docs/configuration/read-command-line-configuration-options-from-files), this gives us a default configuration when running `bundle exec rspec` from the root of the project.

The `--require` flag means we do not need to explicitly mention `require 'spec_helper'` in each test file.